include/glow/Backends/CompiledFunction.h

```diff
@@ -16,11 +16,25 @@
 #ifndef GLOW_BACKENDS_COMPILEDFUNCTION_H
 #define GLOW_BACKENDS_COMPILEDFUNCTION_H
 
+#include <glow/Graph/Nodes.h>
 #include <unordered_map>
 
 namespace glow {
 
 class Context;
+class Placeholder;
+
+struct runtimeSymbolInfo {
```

Comment:
Doxygen please.

Reply:
We use the "_" suffix for class members unless they are really simple pods like (point.x  point.y).

Reply:
Type names in our sources typically start with an upper-case letter.

Please provide comments for the type and all of its fields/methods.

Reply:
I'm also not quite happy about the name of the type. It sounds too general to me. May be use something more specific? And/or may be it should be in its own namespace, e.g. `glow::runtime` or `glow::execution`?

Reply:
I like the glow::runtime namespace, I moved it in there and renamed it to RuntimeBundle and RuntimeSymbolInfo.